CHANGES.current

```diff
@@ -7,6 +7,9 @@ the issue number to the end of the URL: https://github.com/swig/swig/issues/
 Version 4.1.0 (in progress)
 ===========================
 
+2021-05-11: adr26
+            #1985 [Python] Fix memory leaks.
```

Comment:
Need some more detail here please.

Reply:
Please see updated changes for full description of changes.